Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM node:10-alpine AS base
+
+WORKDIR /app
+
+# Dependencies
+COPY package*.json ./
+RUN npm install
+
+# Build
+WORKDIR /app
+COPY . .
+RUN npm run build
+
+# Application
+FROM node:12-alpine AS application
```

Comment:
I would recommend using the same versions of nodejs. F.e. npm version can be changed too or can be used some features from node v12 during the build stage.

Reply:
Yeah, I agree with you. I deployed a tested version, I redeployed now. Thank you that you notice this.